.github/workflows/check-sphinx-and-links.yml

```diff
@@ -29,6 +29,6 @@ jobs:
       id: lc
       uses: lycheeverse/lychee-action@v1.0.8
       with:
-        args: --verbose **/*.html
+        args: --verbose --exclude-mail **/*.html
```

Comment:
Should we maybe remove this now as it had no effect?

Reply:
:heavy_check_mark: